src/logic/operate.js

```diff
@@ -1,20 +1,20 @@
 import Big from 'big.js';
 
 const operate = (numberOne, numberTwo, operation) => {
-  const num1 = new Big(numberOne);
-  const num2 = new Big(numberTwo);
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
   switch (operation) {
     case '+':
-      return num1.plus(num2);
+      return num1.plus(num2).toString();
     case '-':
-      return num1.minus(num2);
+      return num1.minus(num2).toString();
     case 'X':
-      return num1.times(num2);
+      return num1.times(num2).toString();
     case '÷':
-      if (num2 === 0) { return '∞'; }
-      return num1.div(num2);
+      if (num2.eq(0)) { return '∞'; }
+      return num1.div(num2).toString();
     case '%':
-      return num1.mod(num2);
+      return num1.mod(num2).toString();
```

Comment:
this shoud be the percentage of the num1 only, 
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png) 
so basically num1 / 100 should work perfectly